src/components/sbb-tooltip-trigger/sbb-tooltip-trigger.stories.js

```diff
@@ -0,0 +1,85 @@
+import { h } from 'jsx-dom';
+import readme from './readme.md';
+
+const ariaLabel = {
+  control: {
+    type: 'text',
+  },
+};
+
+const iconName = {
+  control: {
+    type: 'text',
+  },
+};
+
+const defaultArgTypes = {
+  'aria-label': ariaLabel,
+  'icon-name': iconName,
+};
+
+const defaultArgs = {
+  'aria-label': undefined,
+  'icon-name': 'circle-information-small',
+};
+
+const tooltip = () => (
+  <sbb-tooltip data-testid="tooltip" trigger="tooltip-trigger">
+    <span id="tooltip-content" class="sbb-text-s">
+      Simple information tooltip with link.
+    </span>
+  </sbb-tooltip>
+);
+
+const Template = (args) => [
+  <span class="sbb-text-s" style="display: flex; align-items: center">
+    <span style="margin-inline-end: var(--sbb-spacing-fixed-1x)">This is a demo text.</span>
```

Comment:
I'm not sure if this margin inline end is really wanted by design. @mcilurzo Can you give a feedback here, or make some examples also in Figma? Also should a dot be before or after the trigger?

Reply:
Margin is wanted, just vertically should be centered also if icon is smaller than line-height (display flex on button probably)